sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -105,6 +105,14 @@ using RT = cl::sycl::detail::pi;
 // Note! This function relies on the fact that all SYCL interface classes
 // contain "impl" field that points to implementation object. "impl" field
 // should be accessible from this function.
+//
+// Note that due to a bug in MSVC compielrs (including MSVC2019 v19.20), it
```

Comment:
Slava, please fix the typo:
compielrs->compilers

Reply:
Fixed. Thank you.